tree.go

```diff
@@ -204,4 +204,6 @@ var (
 	torrent = newMIME("application/x-bittorrent", ".torrent", matchers.Torrent)
 	cpio    = newMIME("application/x-cpio", ".cpio", matchers.Cpio)
 	tzif    = newMIME("application/tzif", "", matchers.TzIf)
+	p7s     = newMIME("application/pkcs7-signature", ".p7s", matchers.P7s).
+		alias(".pem", ".der")
```

Comment:
`alias` is used to set MIME type aliases, not file extensions.  `application/pkcs7-signature` has no aliases so you can remove this line.